chart/open-feature-operator/doc.yaml

```diff
@@ -0,0 +1,54 @@
+## @section Global
```

Comment:
Given that the `value.yaml` file is manually maintained, we might want to have the documentation next to the value. WDYT @toddbaert?

Reply:
I would prefer having values and docs separated in 2 files, at least I find it cleaner. But sure if the team decides it should be together I will adapt it

Reply:
If we have the option, I suggest having the docs as close to the code as possible, i.e. have the docs in the values file directly. It will just make it easier for people to not forget about adding the docs

Reply:
moved docs comments to values.yaml